packages/flutter/test/material/dropdown_menu_test.dart

```diff
@@ -882,6 +918,94 @@ void main() {
     final Material itemMaterial = tester.widget<Material>(buttonMaterial);
     expect(itemMaterial.color, themeData.colorScheme.onSurface.withOpacity(0.12));
   });
+
+  testWidgets('The default text input field should not be focused on mobile platforms '
+      'when it is tapped', (WidgetTester tester) async {
+    final ThemeData themeData = ThemeData();
+
+    Widget buildDropdownMenu({bool? requestFocusOnTap}) => MaterialApp(
```

Comment:
The requestFocusOnTap parameter doesn't appear to be needed here.  Hopefully we have a test that proves that one can override the default value of requestFocus for mobile (to true) and still get the dropdown menu.

Reply:
Removed the parameter! The last test is to override the default value for all platforms to check if the dropdown menu still works. Just added some more checks in that test to make sure we can make a selection when `requestFocusOnTap` is true for mobile or `requestFocusOnTap` is false for desktop.